mandatory/3-function-output.js

```diff
@@ -11,6 +11,9 @@ function combine2Words(word1, word2) {
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  let firstConcat =firstWord.concat(' '+ secondWord)
```

Comment:
At the top of this file there is an extra instruction in the comments. // "Add comments to explain what this function does. You're meant to use Google!" I believe you are supposed to add your own comment there to describe the function.

Reply:
Thank your for your guidance. I changed it

Reply:
I added some comments for functions.